packages/flutter_tools/test/general.shard/runner/flutter_command_test.dart

```diff
@@ -30,6 +30,7 @@ import 'package:test/fake.dart';
 import 'package:unified_analytics/src/enums.dart';
 import 'package:unified_analytics/unified_analytics.dart';
 
+import '../../../lib/src/version.dart';
```

Comment:
```suggestion
import '../../src/version.dart';
```

and sort

Reply:
Good catch, but I think it should be `import 'package:flutter_tools/src/version.dart';`?